### DISCUSSION TITLE
Different default language for geo areas

### DISCUSSION THREAD

**Discussion by CEDEssedi** [globaleaks/globaleaks-whistleblowing-software#4102] [Q&A] [answered]
Hello everyone,

I would like to know if it is possible to set a parameter in the link to globalaks (e.g. ?lang=es) to ensure that the language displayed is the one indicated in the link and not the system default.

In my case I would need to have a link for Italian, one for Spanish and one for English.

At this moment, if a no Italian person opens the link, the page will open in Italian and you will have to "manually" select the preferred language.

You can help me.

Thank you

**Comment by evilaliv3** [comment, marked as answer]
Thank you for your question.

We confirm that as long that you enable Italian, Spanish and English you could use lang=it, lang=es, lang=en respectively

**Reply by CEDEssedi** [reply]
Buongiorno @evilaliv3 

thanks for replying.

Greetings
